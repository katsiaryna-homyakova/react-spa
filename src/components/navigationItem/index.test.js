import React from "react";
import NavigationItem from "./index";
import { shallow } from "enzyme";

describe("NavigationItem component", () => {
  it("renders correctly", () => {
    const component = shallow(<NavigationItem />);
    expect(component).toMatchSnapshot();
  });
});
