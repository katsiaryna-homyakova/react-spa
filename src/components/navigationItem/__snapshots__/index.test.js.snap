// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationItem component renders correctly 1`] = `ShallowWrapper {}`;
